README.md

```diff
@@ -89,6 +89,7 @@ You can use `web3.js` as follows:
 
 ```typescript
 import Web3 from 'web3';
+import { BlockHeader, Block } from 'web3-eth' // ex. package types
```

Comment:
I don't understand the reason or benefit of changing this example. If you think an example for other packages is good to have, then would suggest to create a separate example for it.